syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

message BailmentReply {
    optional uint32 version = 1;
    optional string instructions = 2;
}
